include/lemlib/devices/gyro/gyro.hpp

```diff
@@ -67,14 +69,14 @@ class Gyro {
          * @param update whether to update the last angle measured by the gyro. True by default
          * @return change in angle rotated by the encoder, in radians
```

Comment:
```suggestion
         * @return Angle change in angle rotated by the encoder
```
Update doxygen comment